pkg/appgw/internaltypes.go

```diff
@@ -42,9 +44,9 @@ type secretIdentifier struct {
 	Name      string
 }
 
-const (
-	agPrefix = "k8s-ag-ingress"
-)
+// Max length for a property name is 80 characters. We hash w/ MD5 when length is > 80, which is 32 characters
+var agPrefixValidator = regexp.MustCompile(`^[0-9a-zA-Z\-]{0,47}$`)
```

Comment:
Cool regex!

Reply:
I'll admit - I think adding the validator regex might be a bit of an overkill... But I'd like to make sure that we apply some sanitization a) for security and b) for proactively warning the user when a key does not make much sense.  So it might be a bit of YAGNI - but adds a layer of security just in case.

Reply:
Yes, makes sense.